.github/workflows/release.yml

```
@@ -24,6 +24,10 @@ jobs:
       - uses: actions/checkout@v4
         with:
           persist-credentials: false
+          fetch-depth: '0' # Needed for verify-release-ancestry.sh to see origin/main
+
+      - name: verify release ancestry
```

nit: this is the only step whose name isn't capitalized
```suggestion
      - name: Verify release ancestry
```